.github/workflows/publish.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish
+
+on:
+  push:
+    branches: # Snapshots
+      - main
+    tags: ["*"] # Releases
+
+jobs:
+  publish-artifacts:
+    name: Publish / Artifacts
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+
+      - name: Set up JDK 8
+        uses: olafurpg/setup-scala@v13
+        with:
+          java-version: adopt@1.8
```

Comment:
https://github.com/olafurpg/setup-scala/issues/49 👀  Should I use setup-java?

Reply:
I think we should try switch to [coursier/setup-action](https://github.com/coursier/setup-action)

Reply:
or https://github.com/playframework/play-json/pull/705